sdk-node/src/workflows/workflow.ts

```diff
@@ -485,9 +488,39 @@ export class Workflow<TInput extends WorkflowInput, name extends string> {
       },
     };
 
+    const llm = {
+      structured: <T extends z.ZodObject<any>, TOutput = z.infer<T>>({
```

Comment:
Unless there is a reason now to, I would rather just return the L1M client to keep the signature consistent?